src/services/serverApi.ts

```diff
@@ -195,47 +196,47 @@ export const useGetLevels = () => {
 };
 
 export const filteredTagsData = (
-    tags: tagsWithLevelType[],
-    skills: skills[],
-    searchSkill: string
-  ) => {
-    if (searchSkill !== "") {
-      return tags?.filter((tag) =>
-        tag.name.toLowerCase().includes(searchSkill.toLowerCase())
-      );
-    } else if (skills?.length >= 0) {
-      return tags?.filter(
-        (tag) =>
-          !skills?.some(
-            (skill) =>
-              skill.tagId === tag.tagId && skill.levelId === tag.levelId
-          )
-      );
-    }
-    return tags;
-  };
-
-  export const useUpdateUsersSKillMutation = () => {
-    const reduxDispatch = useDispatch<any>();
-    const [ addNewSkill ] = useAddNewSkillMutation();
-
-    function updateUserSkill(payload: updateSkills) {
-      reduxDispatch(
-        tagsApi.util.updateQueryData('getSkills', payload.itemId, (draft) => {
-          draft?.skills?.push(payload);
-        })
-      )
-
-      addNewSkill({
-        itemId: payload.itemId,
-        itemType: 'USER',
-        tagPayload: [
-          {
-            tagId: payload.tagId,
-            levelId: payload.levelId
-          }
-        ]
+  tags: tagsWithLevelType[],
+  skills: skills[],
+  searchSkill: string
+) => {
+  if (searchSkill !== "") {
+    return tags?.filter((tag) =>
+      tag.name.toLowerCase().includes(searchSkill.toLowerCase())
+    );
+  } else if (skills?.length >= 0) {
+    return tags?.filter(
+      (tag) =>
+        !skills?.some(
+          (skill) =>
+            skill.tagId === tag.tagId && skill.levelId === tag.levelId
+        )
+    );
+  }
+  return tags;
+};
```

Comment:
- Suggestion: This can be two independent if statements. 
```js
if (searchSkill !== "") {
  ...
  return
}

if (skills?.length >= 0) { 
  ....
  return 
}

return tags

```

Reply:
Updated it as per the suggestion, thank you